lua/net.lua

```diff
@@ -0,0 +1,26 @@
+local ffi = require("ffi")
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- TODO Edit the network configuration data.
+local net_config = {
```

Comment:
Drop this table and assign the parameters directly to variable `conf` in `M.setup_block()`.